RELEASE-ANNOUNCEMENT.md

```diff
@@ -51,7 +51,7 @@ You can either download the source code or fork the [Github repository](https://
 Thanks to everyone who did their part to make this release happen:
 - Code contributors: @ann0see, @comradekingu, @danryu, @declension, @hoffie, @ignotus666, @jujudusud, @mcfnord, @pljones, @Rob-NY, @softins
 
-- Application translators: @adoniasalmeida, @ann0see, @BLumia, @bugith, @C10udburst, @comradekingu, @cup113, @dzpex, @fnogcps, @gallegonovato, @gbonaspetti, @gnu-ewm, @ignotus666, @jerone, @mapi68, @MarongHappy, @melcon, @pljones, @softins, @SoulGI, @trebmuh, @Wdtwsm777, @yangyangdaji, @henkdegroot, @ignotus666, Iva Hristova
+- Application translators: @adoniasalmeida, @ann0see, @BLumia, @bugith, @C10udburst, @comradekingu, @cosas, @cup113, @dzpex, @fnogcps, @gallegonovato, @gbonaspetti, @gnu-ewm, @ignotus666, @jerone, @jujudusud, @mapi68, @MarongHappy, @melcon, @pljones, @softins, @SoulGI, @trebmuh, @Wdtwsm777 (Wdt@.+_-), @yangyangdaji, @henkdegroot, @ignotus666, @Skarvinius, Iva Hristova, Kingo Bingo, Katja Flinzner, Gico2006, Massimo Pissarello, Zer0-Zer0
```

Comment:
Ok. Synced with changelog. 

The changelog would now have the content:
```
  * Chinese, by @yangyangdaji, @cup113, @BLumia, @SoulGI, @MarongHappy, Kingo Bingo
  * Dutch, by @henkdegroot, @jerone
  * French, by @trebmuh, @jujudusud, @cosas
  * German, by @ann0see, Ettore Atalan, Iva Hristova, Katja Flinzner
  * Italian, by @dzpex, Gico2006, Massimo Pissarello
  * Korean, by @MarongHappy
  * Norwegian Bokmål, by @comradekingu, Wdt@.+_-
  * Polish, by @gnu-ewm, @C10udburst
  * Portuguese Brazilian, by @melcon, @gbonaspetti, @adoniasalmeida, Zer0-Zer0, @fnogcps
  * Portuguese European, by @melcon
  * Spanish, by @ignotus666, gallegonovato
  * Swedish, by Skarvinius
 ```
  Not found on list above:
  @ bugith, @ mapi68, @ pljones, @ softins, Augusto Kakuja Santos (website)